docker-compose.yml

```diff
@@ -0,0 +1,18 @@
+version: "3.9"
+services:
+  rapid:
+    build:
+      context: .
+      target: scratch
+      args:
+        RAPID_GIT: /rapid-git
+        RAPID_PACKAGES: /rapid-packages
+        RAPID_GIT_REPOS: https://github.com/springraaar/metal_factions
```

Comment:
Is it a leftover/example or is it intended?

Reply:
This is for testing only, I don't intend to merge this PR until it's on a deployable state. mf is smaller than byar for testing